lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -135,6 +135,7 @@ sub engine_workit {
     log_info(sprintf("start time: %s", strftime("%F %T", gmtime)), channels => 'autoinst');
     log_info(sprintf("running on $hostname:%d ($sysname $release $version $machine)", $instance),
         channels => 'autoinst');
+    log_info("isotovideo interface: $OpenQA::Worker::Common::isotovideo_interface_version", channels => 'autoinst');
```

Comment:
My only objection here is that this is just the interface/testapi version, and not the isotovideo version. And both are different things. calling isotoviodeo::version() here would be the saner approach IMO.

Reply:
It would say what it prints: the interface version. The actual isotovideo version could also be added of course.

Reply:
closed.
Related:  https://github.com/os-autoinst/os-autoinst/pull/939